schemas/2019-09-16/Microsoft.HealthcareApis.json

```diff
@@ -0,0 +1,336 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2019-09-16/Microsoft.HealthcareApis.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.HealthcareApis",
+  "description": "Microsoft HealthcareApis Resource Types",
+  "resourceDefinitions": {
+    "services": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "minLength": 3,
+          "maxLength": 24,
+          "description": "The name of the service instance."
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.HealthcareApis/services"
+          ]
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2018-08-20-preview",
```

Comment:
I think this is the reason for your test failure - this API version is defined in multiple places, so the schema parser cannot uniquely identify the resource.

This should just reference a single API version - i.e:

```
        "apiVersion": {
          "type": "string",
          "enum": [
            "2019-09-16"
          ]
        },
```